src/test/java/org/mule/module/xml/validator/SchemaValidationTestCase.java

```diff
@@ -62,6 +61,14 @@ public void invalidSchema() throws Exception {
     validate(getInvalidPayload(), SIMPLE_SCHEMA);
   }
 
+  @Test
+  public void schemaWithReferences() throws Exception {
```

Comment:
As far as I remember, the test for imports here is named featuring include. We should rename that and perhaps this one too to use include.